main.go

```diff
@@ -22,6 +22,7 @@ func main() {
 	// setting up the DAO functions
 	getSourceDao = getSourceDaoWithTenant
 	getApplicationTypeDao = getApplicationTypeWithoutTenant
+	getSourceTypeDao = getSourceTypeWithoutTenant
```

Comment:
This is a mistake on my part, can we rename these functions like `getSourceTypeDAOWithoutTenant` for both the SourceType and ApplicationType DAO?